### DISCUSSION TITLE
in the browser http://localhost:10101/tour I see only Wave Live logo

### DISCUSSION THREAD

**Discussion by THESHAMIK** [h2oai/wave#1434] [General]
Hi, Everything is working for me for Wave scripts.
But when I am running wave apps, it not working.
Web server started-no issues.
But in the browser http://localhost:10101/tour I see only Wave Live logo only.
Similar to A Tour and Demo of Wave doesn't work (VPN/Proxy related)
#725
OR https://github.com/h2oai/wave/discussions/1091.
I am using Windows 11, python 3.7 & Anaconda.
Even a basic prog is not working like:
from h2o_wave import Q, ui, app, main
from h2o_wave import site

@app('/')
async def serve(q: Q):
    await q.page.drop()
    site = q.site
    q.page['random'] = ui.form_card(box='1 12 5 2', items=[ui.progress('Running...')])
    await q.page.save()

There is no error at all. This is what I get after starting the webserver 👍 

(aqua_h2o) C:\Users\shamik.bhattacharya\wave>wave run app.py
params: []2022/05/19 19:29:14 #
2022/05/19 19:29:14 # ┌────────────────┐ H2O Wave
2022/05/19 19:29:14 # │  ┐┌┐┐┌─┐┌ ┌┌─┐ │ 0.21.1 20220512131635
2022/05/19 19:29:14 # │  └┘└┘└─└└─┘└── │ © 2021 H2O.ai, Inc.
2022/05/19 19:29:14 # └────────────────┘
2022/05/19 19:29:14 #
2022/05/19 19:29:14 # {"address":":10101","base-url":"/","t":"listen","web-dir":"D:\\Anaconda3\\envs\\aqua_h2o\\www"}
[32mINFO[0m:     Will watch for changes in these directories: ['C:\\Users\\shamik.bhattacharya\\wave']
[32mINFO[0m:     Uvicorn running on [1mhttp://127.0.0.1:8000[0m (Press CTRL+C to quit)
[32mINFO[0m:     Started reloader process [[36m[1m22444[0m] using [36m[1mstatreload[0m
[32mINFO[0m:     Started server process [[36m20092[0m]
[32mINFO[0m:     Waiting for application startup.
[32mINFO[0m:     Application startup complete.

But why in browser its only wave logo?
Pl help.

**Comment by mtanco** [comment]
Hi @THESHAMIK, can you confirm that `Application startup complete` is the last line in your logs? 

When the location in the your `@app("/")` is visited, we would expect to see the following lines in the logs which show the cards getting added to the screen and the `POST` which is a tab actually viewing the app. 

```log
INFO:     Application startup complete.
2022/05/19 09:12:41 # {"addr":"[::1]:65032","route":"/","t":"ui_add"}
2022/05/19 09:12:41 # {"addr":"[::1]:65032","route":"/cf720df0-c214-4a67-9f72-de25c7a8623c","t":"ui_add"}
INFO:     127.0.0.1:65033 - "POST / HTTP/1.1" 200 OK
2022/05/19 09:12:41 * /cf720df0-c214-4a67-9f72-de25c7a8623c {"d":[{"k":"random","d":{"view":"form","box":"1 1 5 2","items":[{"progress":{"label":"Running..."}}]}}]}
```

Can you confirm that if you're using `@app("/")` that you are specifically going to the `localhost:10101/` URL and not some other path (like `localhost:10101/test`)?


Also, I did change the box in your example from `1 12 5 2` to `1 1 5 2` since starting at a heigh of grid-12 means that the card will be below the bottom of the page:

```python
from h2o_wave import Q, ui, app, main

@app('/')
async def serve(q: Q):
    q.page['random'] = ui.form_card(
        box='1 1 5 2',
        items=[ui.progress('Running...')]
    )
    await q.page.save()
```
